.github/workflows/publish-on-tag.yml

```diff
@@ -0,0 +1,30 @@
+name: publish-on-tag
+
+on:
+  push:
+    tags:
+    - '*'
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install dependencies
+        run: npm install
+      - name: Build
+        run: npm run build
+      - name: Publish puppeteer
+        env:
+          NPM_TOKEN: ${{secrets.NPM_TOKEN_PUPPETEER}}
```

Comment:
Note: I’ve already added `NPM_TOKEN_PUPPETEER` and `NPM_TOKEN_PUPPETEER_CORE` as secrets to the repo.